docs/functions.rst

```diff
@@ -1332,6 +1332,58 @@ Example:
           snapshotClass: do-block-storage
 
 
+CreateCSISnapshotStatic
+-----------------------
+
+This function creates a pair of CSI VolumeSnapshot and VolumeSnapshotContent.
```

Comment:
This seems a bit confusing, does this just create the resource or it also creates the actual volume snapshot. This is explained in the next sentence but I think can be improved.

Reply:
It only provisions the pair of Kubernetes resources. User is required to create the actual underlying snapshot. I think by mentioning "CSI", it's clear that we are talking about Kubernetes API, not the actual storage.

Reply:
I think the place I am getting confused at is, when you say provisioning of Kubernetes resources :). I think `creates` is good, but we should specify that its just the Kubernetes objects that are being created and not actually the volume snapshots.